### DISCUSSION TITLE
/posts/nocode_lowcode_part1

### DISCUSSION THREAD

**Discussion by rjachuthan** [rjachuthan/riturajwrites#3] [Announcements]
In this blog post, let's dive into the world of Low-Code/No Code Data Engineering. But before we do, let's set the stage with a bit of context on the Data Engineering Process.

**Comment by priiyankaaaaaa** [comment]
Very well explained :)

**Reply by rjachuthan** [reply]
Thank you very much @priiyankaaaaaa
